CMakeLists.txt

```diff
@@ -89,7 +89,8 @@ find_package (Unwind)
 
 if (Unwind_FOUND)
   set (HAVE_LIB_UNWIND 1)
-  set (HAVE_UNWIND_H 1)
+else (Unwind_FOUND)
+  check_include_file_cxx (unwind.h HAVE_UNWIND_H)
```

Comment:
Why do you need this? `FindUnwind.cmake` already accomplishes this.

Reply:
`FindUnwind.cmake` is for libunwind library. It has `libunwind.h` and `libunwind.so`. And its API is `unw_*` and is low level. `unwind.h` contains high level API `_Unwind_*` (based on `unw_*` in case of [LLVM](https://github.com/llvm/llvm-project/tree/main/libunwind/include)).

Reply:
Please check #629 for a related discussion before making further adjustments.

Reply:
`HAVE_LIB_UNWIND` includes `libunwind.h` and `HAVE_UNWIND_H` includes `unwind.h`. They are irrelevant in current code:

https://github.com/google/glog/blob/9733727be4aed835e21a8f73c403f546b71663fe/src/utilities.h#L108-L114

Reply:
Though Android NDK uses LLVM, only `unwind.h` is shipped.